front/src/modules/trainschedule/components/ManageTrainSchedule/ManageTrainScheduleMap/AddPathStepPopup.tsx

```diff
@@ -73,33 +67,25 @@ function RenderPopup({ pathProperties }: RenderPopupProps) {
       setTrackOffset(offset);
     };
 
-    if (featureInfoClick.displayPopup) {
+    if (featureInfoClick) {
```

Comment:
You know that featureInfoClick is defined now ;)
```suggestion
```

Reply:
Done